src/lua/elrsV2.lua

```diff
@@ -801,23 +820,51 @@ local function handleDevicePageEvent(event)
       end
     end
   elseif edit then
-    if event == EVT_VIRTUAL_NEXT then
-      incrField(1)
-    elseif event == EVT_VIRTUAL_PREV then
-      incrField(-1)
+    if event ~= nil then
+      if event ~= 0 then
+        if touchState then
+          if event == EVT_TOUCH_SLIDE then
+            if touchState.swipeUp then
+              incrField(1)
+            elseif touchState.swipeDown then
+              incrField(-1)
+            end
+          end
+        else -- event ~= 0 and touchState == nil: key event
+          if event == EVT_VIRTUAL_NEXT then
+            incrField(1)
+          elseif event == EVT_VIRTUAL_PREV then
+            incrField(-1)
+          end
+        end
+      end
     end
   else
-    if event == EVT_VIRTUAL_NEXT then
-      selectField(1)
-    elseif event == EVT_VIRTUAL_PREV then
-      selectField(-1)
+    if event ~= nil then
+      if event ~= 0 then
+        if touchState then
+          if event == EVT_TOUCH_SLIDE then
+            if touchState.swipeLeft then
+              selectField(-1)
+            elseif touchState.swipeRight then
```

Comment:
and this to .swipeUp? so swipe up/down is acting like the scroll behaviour

Reply:
Please have a look at [905f096](https://github.com/ExpressLRS/ExpressLRS/pull/1515/commits/905f096594f43acd014a0fac0a0a50b65665b219)